style.css

```diff
@@ -0,0 +1,97 @@
+/* This is a commnet in CSS */
+
+body {
+  background-image: url("https://images.unsplash.com/photo-1519751138087-5bf79df62d5b?ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80");
+  background-color: #000000;
+  background-position: left top, top;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+h1{
+    color: navy;
+    margin-left: 20px;
+  }
+/* div p {
+    color: blanchedalmond;
+    background-color: orange;
+   
+    width: auto;
+    border: 1px solid whitesmoke;
+    padding: 50px;
+    margin: 20px;
+}
+*/
+.banner.note {
+
+}
+
+.banner {
+    /* color: blanchedalmond;
+    background-color: orange; */
+    /* Boarders & Boxes */
+    width: auto;
+    border: 1px solid whitesmoke;
+    padding: 50px;
+    margin: 20px;
+
+}
+/*
+*/
```

Comment:
/* */ and // and # comments don't work in html.  The way to do comments in HTML is through ```<!-- stuff being commented out -->```.  Some languages have a semi-HTML syntax such as ERB, Razor, ..., which will allow you to do that in certain block selections such as:
```razor
<div id="example">
    @foreach (var thing in StuffList) {
        //Rendering multiple elements here
        <p>@thing.Value</p>
       /* Here we will render the same element for each thing
        * this is because each thing needs a close button
        */
        <button type="button">Close</button>
    }
</div>
```

The reason these semi-HTML languages have this is because they render the code inline allowing you to write display code on the same page as the HTML for readability purposes.  Therefore, to comment code, we wouldn't use the HTML comment markup.  I don't think this applies here though.